contributing-docs/11_provider_packages.rst

```diff
@@ -117,6 +117,28 @@ as a sub-folder of the ``airflow.providers`` package, add the ``provider.yaml``
 in development mode - then capabilities of your provider will be discovered by airflow and you will see
 the provider among other providers in ``airflow providers`` command output.
 
+
+Local Development Release of a Specific Provider
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you develop a provider, you can release it locally and test it in your Airflow environment. This should
+be accomplished using breeze. First, choose a version number that will never be used in an official release.
```

Comment:
You can also use `--skip-tag-check` if you want to release already released version.  I think it's worth describing here.

Reply:
oh, good to know. Thanks!